Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -4148,3 +4150,45 @@ bool CLuaVehicleDefs::BlowVehicle(CClientEntity* entity, std::optional<bool> wit
 {
     return CStaticFunctionDefinitions::BlowVehicle(*entity, withExplosion);
 }
+
+std::variant<bool, std::vector<std::array<float, 3>>> CLuaVehicleDefs::GetVehicleEntryPoints(CClientVehicle* vehicle)
+{
+    if (CClientVehicleManager::GetMaxPassengerCount(vehicle->GetModel()) == 255)
+    {
+        return false;
+    }
+
+    std::vector<std::array<float, 3>> entryPoints;
```

Comment:
If you already know the size of the vector, why use a vector?
```cpp
std::array<std::array<float, 3>, 4>
```
Would be much better.